src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -27,16 +27,16 @@ public void OverflowOnFluentCall()
 
             switch (IntPtr.Size * 8) {
                 case 32 when isDebug:
-                    numberFluentCalls = 290;
+                    numberFluentCalls = 510;
```

Comment:
Why is Debug so much smaller?  Is it because of our code or limitations in the JIT?